### PR TITLE
feat: add get_card_attachments and get_card_checklists tools

### PR DESCRIPTION
**Pull request by sandikodev**
﻿## Description

Add two new MCP tools for retrieving card data from Trello:

### `get_card_attachments`
Get all attachments from a specific card. Returns file name, URL, MIME type, size, and metadata for each attachment.

### `get_card_checklists`
Get all checklists on a card with their items, including completion percentage for each checklist. Returns structured data with item names, checked state, and position.

## Changes
- `src/trello/attachments.ts` — Add `getCardAttachments` function
- `src/trello/checklists.ts` — New file with `getCardChecklists` function
- `src/trello-client.ts` — Add `getCardAttachments` and `getCardChecklists` methods
- `src/index.ts` — Register both tools with Zod input schemas

## Testing
Both tools tested against Trello API and confirmed working.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+﻿#!/usr/bin/env node
```
[the remark this answers is not in the document]
Comment (sandikodev) [reply src/index.ts]: i've commit `--amend --no-edit` now, i just Fixed. removed UTF-8 BOM (so sorry, this is my fault on Windows PS 5.1 Set-Content -Encoding UTF8) from the file.

**Comment by sandikodev** [comment]
Fixed — removed the UTF-8 BOM from the shebang line. The file was saved with BOM by PowerShell's Set-Content encoding. Now rewritten without BOM.

**Review by delorenj — APPROVED** [review]
Approved via pr-crusher: new tools cover a real workflow (checklist normalization + completion %), tests added to clear the coverage gate, suite green. Idiomatic and clean.

**Comment by delorenj** [comment]
Love this one, @sandikodev — `get_card_checklists` normalizing raw Trello checklists into the repoʼs `CheckList` DTO with a derived `percentComplete` is exactly the kind of encapsulation that saves every downstream caller from re-implementing the same mapping. 🙌

To get it over our coverage bar I pushed a small test commit to your branch (`maintainerCanModify` was on, so I hope thatʼs welcome): unit tests for both `getCardChecklists` (URL, the full `CheckList` shape mapping, and the `percentComplete` math incl. the empty / undefined `checkItems` edge cases) and `getCardAttachments`, plus two tiny cleanups — dropped an unused `CheckListItem` import and trimmed a couple trailing blank lines in `index.ts`. Suite is green at **114 passed / 26 skipped**.

Merging now — thank you for the clean, focused PR. Heads-up: your other two (#99, #100) touch the same regions of `index.ts` / `trello-client.ts`, so theyʼll want a quick rebase onto `main` after this lands; Iʼll drop a note on each.
